M2/Macaulay2/tests/ComputationsBook/completeIntersections/test.out.expected

```diff
@@ -1,5 +1,6 @@
 
 i1 : setRandomSeed();
+ -- initializing random seed
```

Comment:
Should the `recursionDepth` check be preventing this?

Reply:
I thought so, but somehow it didn't and the tests still failed, so I removed the check.